packages/fluentui/react-northstar/src/components/Tree/Tree.tsx

```diff
@@ -328,103 +336,106 @@ class Tree extends AutoControlledComponent<WithAsProp<TreeProps>, TreeState> {
     elementToBeFocused.focus();
   };
 
-  onSiblingsExpand = (e: React.SyntheticEvent, treeItemProps: TreeItemProps) => {
-    const { exclusive, items } = this.props;
+  const onSiblingsExpand = (e: React.SyntheticEvent, treeItemProps: TreeItemProps) => {
     if (exclusive) {
       return;
     }
 
     const { id } = treeItemProps;
-    const { activeItemIds } = this.state;
+
     const siblings = getSiblings(items, id);
 
+    const nextActiveItemsIds = [...activeItemIds];
+    if (activeItemIds.indexOf(id) === -1) {
```

Comment:
```suggestion
    if (!isActiveItem(id)) {
```
By the way, on lines 359-360 you have similar check, could they be unified?

Reply:
Done